lib/Doctrine/Migrations/InlineParameterFormatter.php

```diff
@@ -53,7 +53,7 @@ public function formatParameters(array $params, array $types): string
         return sprintf('with parameters (%s)', implode(', ', $formattedParameters));
     }
 
-    private function formatParameter(mixed $value, string|int $type): string|int|float|null
+    private function formatParameter(mixed $value, string|int $type): string|int|bool|float|null
```

Comment:
Why this change ? `parameterToString` returns a string.

Reply:
`parameterToString` does, but `Type::getType('string')->convertToDatabaseValue(...)` don't